src/main/java/org/checkerframework/specimin/PrunerVisitor.java

```diff
@@ -4,6 +4,7 @@
 import com.github.javaparser.ast.ImportDeclaration;
 import com.github.javaparser.ast.Node;
 import com.github.javaparser.ast.NodeList;
+import com.github.javaparser.ast.body.AnnotationDeclaration;
```

Comment:
I would not expect spotless (the auto-formatter) to pass with this unused import declaration.

Reply:
I don't think there should be any changes in this file.